gef-ui/server/controllers/mocks/index.js

```diff
@@ -116,7 +116,7 @@ const MockApplicationResponseSubmitted = () => {
     comments: [],
     ukefDealId: 123456,
     createdAt: chance.timestamp(),
-    submissionDate: yesterday,
+    submissionDate: `${getUnixTime(yesterday).toString()}449`,
```

Comment:
What is the constant 449 for please?

Reply:
just a random number to add ms onto submissionDate, else it breaks without milliseconds